draft-ietf-masque-connect-ip.md

```diff
@@ -1150,11 +1150,25 @@ ROUTE_ADVERTISEMENT capsule.
 # Security Considerations
 
 There are significant risks in allowing arbitrary clients to establish a tunnel
-that permits sending to arbitrary hosts, as that could allow bad actors to send traffic and have it
-attributed to the IP proxy. IP proxies SHOULD restrict its use
-to authenticated users. The HTTP Authorization header {{HTTP}} MAY be
-used to authenticate clients. More complex authentication schemes are out of
-scope for this document but can be implemented using extensions.
+that permits sending to arbitrary hosts, regardless of whether tunnels are
+scoped to specific hosts or not. Bad actors could abuse this capability
+to send traffic and have it attributed to the IP proxy. IP proxies SHOULD
+restrict its use to authenticated users. Depending on the deployment,
+possible authentication mechanisms include mutual TLS between clients
+and proxies, HTTP-based authentication via the HTTP Authorization header
```

Comment:
Perhaps we should call out specific header names, since both Authorization and Proxy-Authorization could be used here.

Reply:
I don't think it's necessary, but I defer to y'all as the editors to do whatever you think is best!

Reply:
I propose we keep the current text and wait for IETF LC / IESG review to bikeshed these examples further

Reply:
I would think that proxy-authorization would not be appropriate to use, as the authorization required is related to the HTTP target, not an HTTP proxy before it (for which Prozy-Authorization would be appropriate).